### PR TITLE
CLI 1.19.2 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2834

Looks like upgrade-test is stabilizing and we're using it a lot with customers, so also added that into our upgrade Runtime guide.

### CODE REVIEW

**Comment by jwitz** [comment]
@lzdanski A previous writer thought we should always include the upgrade considerations before the core setup. What do you think about instead keeping those at the bottom of the doc so that users don't have to scroll loads just to see how to upgrade?

**Comment by lzdanski** [comment]
@lzdanski I think that makes sense and then have relative linking so that users can skip down to where they need would work!

So like 

```
## Upgrade considerations
... <existing content>

Each version of Astro runtime introduces new features, behaviors, and bug fixes. Before upgrading to a particular version, check to see if there are any version specific considerations you need to account for:

- Runtime version 9 (link to v9 at bottom)
-  etc
```

And then at the end

```
## Runtime Version considerations

### Version 9 
<add existing content>

```

**Comment by lzdanski** [comment]
@jwitz  I think that makes sense and then have relative linking so that users can skip down to where they need would work!

So like 

```
## Upgrade considerations
... <existing content>

Each version of Astro runtime introduces new features, behaviors, and bug fixes. Before upgrading to a particular version, check to see if there are any version specific considerations you need to account for:

- Runtime version 9 (link to v9 at bottom)
-  etc
```

And then at the end

```
## Runtime Version considerations

### Version 9 
<add existing content>

```

**Review by sunkickr — APPROVED** [review]

**Review by sunkickr — APPROVED** [review]

File: astro/upgrade-runtime.md [hunk 1]
```diff
@@ -13,6 +13,102 @@ import {siteVariables} from '@site/src/versions';
...
+
+:::info
+
+If you're only upgrading a local Airflow environment, you don't need an Astro Deployment and you can skip steps 7-8.
+
+:::
+
+## Step 1: Review upgrade considerations
+
+Astro upgrades can include breaking changes, especially when you're upgrading to a new major version. Check the [upgrade considerations](#upgrade-considerations) for your upgrade version to anticipate any breaking changes or upgrade-specific instructions.
```
Comment (jwitz) [inline astro/upgrade-runtime.md:30]:
```suggestion
Astro upgrades can include breaking changes, especially when you're upgrading to a new major version. Check the [upgrade considerations](#upgrade-considerations) for your upgrade version to anticipate any breaking changes or upgrade-specific instructions before you proceed.
```

**Pull request merged**